examples/plot_meg_inverse_solution.py

```diff
@@ -13,18 +13,16 @@
 
 print(__doc__)
 
-path = os.path.dirname(__file__)
-
 """
 define subject, surface and hemisphere(s) to plot
 """
 subject_id, surf = 'fsaverage', 'inflated'
-hemi = 'split'
+hemi = 'lh'  # 'split'
```

Comment:
I left this in intentionally to "suggest" to the user they could do it, but sure